inspirehep/modules/workflows/workflows/article.py

```diff
@@ -349,12 +349,10 @@
                         [
                             reject_record('Article automatically rejected'),
                             mark('approved', False),
-                            save_workflow,
-                            stop_processing,
                         ],
-                    )
-                )
-            ]
+                    ),
```

Comment:
we don't have to `save_workflow`?

Reply:
we can but we don't have to as wf is not stopped. It's saved literally 3 lines below and this executes for all paths.